Src/Extern/openPMD-api/AMReX_PlotFileUtilOPENPMD_PTLImpl.H

```diff
@@ -0,0 +1,661 @@
+#ifndef AMREX_PLOTFILE_OPENPMD_PTLS_API_H
+#define AMREX_PLOTFILE_OPENPMD_PTLS_API_H
+
+//extern  std::unique_ptr< amrex::openpmd_api::AMReX_openPMDHandler > amrex::openpmd_api::m_OpenPMDHandler;
+namespace amrex::openpmd_api
+{
+    extern  std::unique_ptr< amrex::openpmd_api::AMReX_openPMDHandler > m_OpenPMDHandler;
+    ////////////////////////////////////////////////////////////////////////
+    //
+    // Global function to write particle container
+    //   each WriteParticles() call has two versions:
+    //   with FUNC(for user supploed saveposid) or not (using default from the writer)
+    //
+    ////////////////////////////////////////////////////////////////////////
+
+    template<class PC, typename FUNC_pc, typename FUNC_pit>
+    void WriteParticles(PC const&  pc,
+                        const std::string& specieName,
+                        //int iteration,
+                        const Vector<std::string>& real_comp_names,
+                        const Vector<std::string>& int_comp_names,
+                        const Vector<int>& do_write_real_comp,
+                        const Vector<int>& do_write_int_comp,
+                        FUNC_pc&& pcFunc,
+                        FUNC_pit&& posIdFunc)
+    {
+      BL_PROFILE("amrex::openpmd_api::WriteParticles()");
+
+      amrex::Print()<<"   Real: "<<pc.NStructReal<<" "<<pc.NumRealComps()<<"\n";
```

Comment:
We need to follow the xSDK policy that the user must be able to disable print statements. So could we put them inside `if (amrex::Verbose())`?

Reply:
I will remove these. Was for debugging.